lomap/dbmol.py

```diff
@@ -379,8 +391,8 @@ def ecr(mol_i, mol_j):
             
             for atom in mol_j.GetAtoms():
                 total_charge_mol_j += float(atom.GetProp('_TriposPartialCharge'))
-
-            if abs(total_charge_mol_j - total_charge_mol_i) < 1e-3:
+            #sliu 07/17 change to a loose threshold
+            if abs(total_charge_mol_j - total_charge_mol_i) < 1e-2:
```

Comment:
I think it'll not be changed for most of the time, so it's fine to set a fixed default value here I guess.